[DEVX-956] Add reporters documentation for cypress (#591)
saucelabs/sauce-docs: docs/testrunner-toolkit/configuration/cypress.md

:::
---

### `reporters`
<p><small>| OPTIONAL | OBJECT |</small></p>

The additional reporters you want to add.

```yaml
  reporters:
    - name: cypress-mochawesome
      options:
        reportDir: cypress/report
        charts: true
        reportPageTitle: Cypress running on Sauce
```

:::note
Reporters functionality is provided through [cypress-multi-reporter](https://www.npmjs.com/package/cypress-multi-reporters).
In order for your additional reporter to work, it must be compatible with that plugin.
:::

---

#### `name`
<p><small>| REQUIRED | STRING |</small></p>

> The name of the reporter to enable.

Reviewer: ```suggestion
The name of the reporter to enable, which corresponds to the `reporter` property in the `cypres.json` file.
```

```yaml
      name: cypress-mochawesome
```

> :::note
> This field has the same purpose as `reporter` in `cypress.json` file.
> 
> Some reporters may require you to install dependencies.
> :::

Reviewer: ```suggestion
:::note
Some reporters may require you to install dependencies.
:::
```

---

#### `options`
<p><small>| OPTIONAL | OBJECT |</small></p>

> The options of reporters you want to add.

Reviewer: ```suggestion
Any relevant settings that are be supported by the specified reporter. These properties correspond to the `reporterOptions` object in the `cypress.json` file.
```

```yaml
      options:
        reportDir: cypress/report
        charts: true
        reportPageTitle: Cypress running on Sauce
```

:::note
This field has the same purpose as `reporterOptions` in `cypress.json` file.

> Reporter options are specific for each reporter. Take a look to your reporter options in order to know which options are available.

Reviewer 2: ```suggestion
Reporter options are specific for each reporter. Consult your reporter documentation, in order to know which options are available.
```

> :::note
> This field has the same purpose as `reporterOptions` in `cypress.json` file.
> 
> Reporter options are specific for each reporter. Consult your reporter documentation, in order to know which options are available.
> :::

Reviewer: ```suggestion
```
